app/views/ontologies/browse.html.erb

```diff
@@ -193,9 +193,9 @@
         </div>
       </div>
 
-      <div class="ontology" ng-show="visible_ont_count == 0" style="border: solid thin lightGray;">
+      <div ng-show="visible_ont_count == 0" style="border: solid thin lightGray; margin-top:2em;">
         <div style="text-align: center; color: gray; height: 2em; padding: 4em;">
-          <h2 style="font-weight: normal;">No matches!</h2>
+          <h2 style="font-weight: normal; font-size: 3vw; ">&nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; No Matches! &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</h2>
```

Comment:
i don't think that spaming "&nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; No Matches! &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp" is the solutions.

Think more about padding, layouts (flex, grid, ..)